web/react/stores/preference_store.jsx

```diff
@@ -122,5 +122,6 @@ class PreferenceStoreClass extends EventEmitter {
 }
 
 const PreferenceStore = new PreferenceStoreClass();
+PreferenceStore.setMaxListeners(0);
```

Comment:
The goal is to re-render the posts if preferences change like favourite posts list, or chosen post style. Therefore it is enough if post list wrapper component listens on prefs because it will rerender anyway, and starreditem can just call getPreferences without doing listeners. Would that make sense to you?